website/docs/networking/vpc-lattice/cleanup.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Cleanup"
+sidebar_position: 25
+---
+
+```bash
```

Comment:
These clean-up steps should be significantly less complex once we switch to using kustomization > sed/patch/etc Kustomize will be able to automatically clean up anything not in the original manifests directory. Also, `reset-environment` is expected to automatically roll back any changes done in a module.

After making above changes to use Kustomize, we will also need to update `reset-environment` with idempotent changes to ensure that users can run `reset-environment` at any stage of running through this lab to reset changes.

Reply:
+1 to Sai's take. In any case we should set expectation about the order of these steps and any time delays associated with individual command execution.

Reply:
Simplified the setup/cleanup using kustomize as suggested. I’m not changing the part where we patch the UI configmap and the existing service because it would be a little bit of an overkill. Please, check the new version. With the current cleanup steps the reset-environment should bring everything back to normal from a user point of view.